docs/team/pokezardvgc.md

```diff
@@ -24,4 +24,5 @@ Given below are my contributions to the project.
 
 * **Community**:
 
-* **Tools**:
\ No newline at end of file
+* **Tools**:
+* 
```

Comment:
Having an extra * makes it not an empty newline. Just leave a **blank** line at the end.